packages/admin/cms-admin/src/index.ts

```diff
@@ -1,5 +1,21 @@
 export { AnchorBlock } from "./blocks/AnchorBlock";
+export { readClipboard } from "./blocks/clipboard/readClipboard";
+export { writeClipboard } from "./blocks/clipboard/writeClipboard";
```

Comment:
those are deprecated, maybe you could get rid of them now - as you need a upgrade script anyway

Reply:
Done: fcc2b7f870fb84dd8df35f0380c29a157c414eb0